laplace/baselaplace.py

```diff
@@ -228,8 +260,9 @@ def optimize_prior_precision_base(
             otherwise, the structure of init_prior_prec is maintained.
         val_loader : torch.data.utils.DataLoader, default=None
             DataLoader for the validation set; each iterate is a training batch (X, y).
-        loss : callable, default=get_nll
-            loss function to use for gridsearch.
+        loss : callable or torchmetrics.Metric, default=get_nll
```

Comment:
Should we change the default to the `RunningNLLMetric`?

Reply:
Let's leave this for a new PR